client/src/main/java/io/pravega/client/segment/impl/SegmentInputStreamImpl.java

```diff
@@ -196,7 +196,7 @@ private int computeReadLength(long currentFetchOffset) {
     public void close() {
         log.trace("Closing {}", this);
         if (outstandingRequest != null) {
-            log.trace("Cancel outstanding read request for segment {}", asyncInput.getSegmentId());
+            log.debug("Cancel outstanding read request for segment {}", asyncInput.getSegmentId());
```

Comment:
What about a message after `outstandingRequest.cancel(true)` to detect stalls?

Reply:
added it